x-pack/plugins/cloud_security_posture/server/lib/telemetry/collectors/types.ts

```diff
@@ -8,7 +8,7 @@
 import { CspStatusCode } from '../../../../common/types';
 
 export interface CspmUsage {
-  indices: CspmIndicesStats;
+  indices: CspmIndicesStats | never[];
```

Comment:
From what I've read about `never` it doesn't appear to be related here

my reference: https://www.typescriptlang.org/docs/handbook/basic-types.html#never

Reply:
Okay, good point. I mistakenly thought we wanted to handle the error with an empty state so applying `never` fixed the issue type error but I'll change the default value to `undefined`.